packages/site/cms-site/src/blocks/YouTubeVideoBlock.tsx

```diff
@@ -25,7 +44,7 @@ const parseYoutubeIdentifier = (value: string): string | undefined => {
     return youtubeId ?? undefined;
 };
 
-const YouTubeVideoBlock: React.FunctionComponent<PropsWithData<YouTubeVideoBlockData>> = ({
+export const YouTubeVideoBlock: React.FunctionComponent<PropsWithData<YouTubeVideoBlockData>> = ({
```

Comment:
We shouldn't loose the `withPreview` functionality

Reply:
> What exactly does withPreview do?

It adds the overlay used for selecting/hovering the block in site

> Because in the Admin I still can see a preview 🤔

Probably because you're using the YouTubeVideoBlock in a OneOfBlock which has its own withPreview. But if you were to use it as a standalone block, the preview wouldn't work anymore.